schemas/string_interpolation/transforms/format_unit.json

```
@@ -0,0 +1,31 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/format_unit.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "format_unit": {
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["format_unit"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/placeholder_value_sources"
+          },
+          "length": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+          },
```

This should be an enum as it can only be either `short` or `long` or `narrow`. I think also `unit_length` would be clearer and would match what it is called in answer objects.
```suggestion
          "unit_length": {
            "type": "string",
            "enum": ["short", "narrow", "long"]
          },
```